### PR TITLE
mongodb changes for nodebb

### PR DESCRIPTION
**Pull request by harshavardhanc**

### CODE REVIEW

File: kubernetes/helm_charts/core/nodebb/values.j2 [hunk 1]
```diff
@@ -24,4 +24,4 @@ strategy:
   maxunavailable: {{ nodebb_maxunavailable|default('25%') }}
 
 nodebb_url: {{ sunbird_nodebb_url }}
-redis_nodebb_host: {{ redis_nodebb_host }}
+mongo_nodebb_host: {{ mongo_nodebb_host }}
```
Comment (rjshrjndrn) [inline kubernetes/helm_charts/core/nodebb/values.j2:27]: what's the default value for this ?

Comment (rjshrjndrn) [reply kubernetes/helm_charts/core/nodebb/values.j2:27]: the default entry should be in all.yaml like
mongo_nodebb_host: group['mongo'] | join ','

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: kubernetes/helm_charts/core/nodebb/values.j2 (hunk 1 above)
Comment (harshavardhanc) [reply kubernetes/helm_charts/core/nodebb/values.j2:27]: Updated

**Review by rjshrjndrn — APPROVED** [review]

File: kubernetes/helm_charts/core/nodebb/values.j2 (hunk 1 above)
Comment (rjshrjndrn) [reply kubernetes/helm_charts/core/nodebb/values.j2:27]: will the application support ',' seperated ips ?

File: ansible/inventory/env/group_vars/all.yml [hunk 2]
```diff
@@ -649,3 +649,4 @@ group_activity_agg_cache_enable: false
 
 # nodebb variables
 sunbird_nodebb_storage_key: "{{ core_vault_sunbird_nodebb_storage_key }}"
+mongo_nodebb_host: "{{ groups['mongo'] | join ',' }}"
```
Comment (rjshrjndrn) [inline ansible/inventory/env/group_vars/all.yml:652]: will the app support ',' seperated ips @vinukumar-vs

**Review by rjshrjndrn — CHANGES_REQUESTED** [review]

File: ansible/inventory/env/group_vars/all.yml (hunk 2 above)
Comment (rjshrjndrn) [reply ansible/inventory/env/group_vars/all.yml:652]: https://community.nodebb.org/topic/10810/connection-to-mongodb-that-is-hosted-on-atlas/2

**Review by rjshrjndrn — APPROVED** [review]

**Pull request merged**